components/landing/communication-portal.tsx

```diff
@@ -0,0 +1,157 @@
+'use client'
+
+import { Button } from "@/components/ui/button"
+import { Send, Sparkles } from "lucide-react"
+import Image from "next/image"
+
+export default function CommunicationPortal() {
+  const structuredData = {
+    "@context": "https://schema.org",
+    "@type": "Organization",
+    "name": "The Technologian Student Press",
+    "description": "Official student publication of Cebu Institute of Technology - University",
+    "url": "https://thetechnologian.com",
+    "foundingDate": "1987",
+    "address": {
+      "@type": "PostalAddress",
+      "streetAddress": "N. Bacalso Ave",
+      "addressLocality": "Cebu City",
+      "postalCode": "6000",
+      "addressRegion": "Cebu",
+      "addressCountry": "Philippines"
+    },
+    "contactPoint": {
+      "@type": "ContactPoint",
+      "email": "thetechnologianstudentpress@gmail.com",
+      "contactType": "Editorial"
+    },
+    "sameAs": [
+      "https://facebook.com/thetechnologian",
+      "https://instagram.com/thetechnologian"
+    ]
+  };
+
+  return (
+    <main className="min-h-screen bg-dotted-grid-subtle relative overflow-hidden" role="main">
+      {/* Structured Data */}
+      <script
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{ __html: JSON.stringify(structuredData) }}
+      />
```

Comment:
Move this inside a Next.js Head instead to avoid any possible xss injections from happening
```suggestion
        return (
    <>
      <Head>
        <script
          type="application/ld+json"
          dangerouslySetInnerHTML={{ __html: JSON.stringify(structuredData) }}
        />
      </Head>
      <main className="min-h-screen bg-dotted-grid-subtle relative overflow-hidden" role="main">
        {/* Remove the script tag from here */}
        
        {/* ...existing code... */}
      </main>
    </>
  )
```